docs/src/rust-feature-support.md

```diff
@@ -200,3 +200,14 @@ related to [advanced features](#advanced-features).
 
 Please refer to [Intrinsics](rust-feature-support/intrinsics.md) for information
 on the current support in Kani for Rust compiler intrinsics.
+
+### Floating point operations
+
+Kani supports floating point numbers, but some supported operations on floats are "over-approximated."
+These are the trigonometric functions like `sin` and `cos` and the `sqrt` function as well.
+This means the verifier can raise errors that cannot actually happen when the code is run normally.
+For instance, ([#1342](https://github.com/model-checking/kani/issues/1342)) the `sin`/`cos` functions basically return a nondeterministic value between -1 and 1.
+In other words, they largely ignore their input and give very conservative answers.
+This range certainly includes the "real" value, so proof soundness is still preserved, but it means Kani could raise spurious errors that cannot actually happen.
+This makes Kani unsuitable for verifying some kinds of properties (e.g. precision) about numerical algorithms.
+Proofs that fail because of this problem can sometimes be repaired by introducing "stubs" for these functions that return a more acceptable approximation.
```

Comment:
It may be worth adding a note to the effect of what @martin-cs said in https://github.com/diffblue/cbmc/issues/6999#issuecomment-1220773186: actual implementations may vary from platform to platform, and more precise models (stubs) may well introduce unsoundness.

Reply:
Is there even a standard for numerical accuracy for Rust or is it implementation defined?  This:

https://stackoverflow.com/questions/21177436/is-there-a-published-language-format-standard-for-rust-yet

makes me a little concerned.  I note that:

https://doc.rust-lang.org/stable/reference/types/numeric.html

links to IEEE-754 2008 and that should be correctly supported.  The documentation for the numerical functions is ... a bit thin:

https://doc.rust-lang.org/std/primitive.f64.html#method.sin